openshift-install-powervs

```diff
@@ -1069,6 +1070,50 @@ function setup_ibmcloudcli() {
   ${CLI_PATH} -v
 }
 
+
+#-------------------------------------------------------------------------
+# Install oc cli in current directory
+# If latest is available in System PATH then use symlink
+#-------------------------------------------------------------------------
+function setup_occli() {
+
+  OC_PLATFORM="$OS"
+  OC_BIN="oc.tar.gz"
+
+  if [[ "$OS" == "darwin" ]]; then
+    OC_PLATFORM="macosx"
+  fi
+
+  if [[ "$OS" == "windows" ]]; then
+    OC_BIN="oc.zip"
+  fi
+
+  OC_BASE_URL="https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/oc"
```

Comment:
This location is deprecated [1]. We can use the variable `openshift_client_tarball` if provided by the user and replace `linux` with the client machine OS.

[1] https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/oc/4.6/$__DEPRECATED_LOCATION__PLEASE_READ__.txt

Reply:
yeah right. I'll make the changes to point to the updated location